target/linux/ath79/image/nand.mk

```diff
@@ -1,3 +1,16 @@
+define Build/dw-headers
+	head -c 4 $@ >> $@.tmp && \
```

Comment:
For the first command, probably ">" would be better than ">>"?

Reply:
Probably right, I'll prepare a tidy-up patch with that change.